websites/W/Warriorcats/presence.ts

```diff
@@ -0,0 +1,270 @@
+const presence = new Presence({
+		clientId: "936630095035113522",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+async function getPostNumber() {
+	// find <article></article> and get the #post- number
+	let postNumber = document.querySelector("article").getAttribute("id");
+	postNumber = postNumber.replace(/post-/, "");
+	return postNumber;
```

Comment:
This code (and other functions) can be simplified to a single `return` statement.